sale_forecast/README.rst

```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Sale Forecast
+==============
+
+Based on `procurement_sale_forecast <https://github.com/odoomrp/odoomrp-wip/tree/8.0/procurement_sale_forecast>`_.
+
+This module allows you to create a sale forecast.
+
+* Possibility to load sale order lines or a sale forecast previously created.
+* Allows you to group forecast lines by product, partner, date, category and forecast.
+* There's a pivot view where you can see the real quantity and the forecasted quantity.
+
+Usage
+=====
+
+Sales -> Sale Forecast
+
+Contributors
+============
+
+* Carlos Martínez <carlos@domatix.com>
+* Nacho Serra <nacho.serra@domatix.com>
+* Nacho HM <nacho@domatix.com>
+* Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
+* Ainara Galdona <ainaragaldona@avanzosc.es>
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```

Comment:
Please change @pedrobaeza email to pedro.baeza@tecnativa.com

Reply:
There are no previous commits cause this module is slightly based on "procurement_sale_forecast" but the code is quite different as well as the functionality. We decided to keep the original autors as contributors, but they might be moved to another section, such as "Module inspired in "procurement_sale_forecast" developed by "(original authors)"...".